cmd/krel/cmd/changelog.go

```diff
@@ -89,16 +90,17 @@ func init() {
 	cobra.OnInitialize(initConfig)
 
 	const (
+		tagFlag   = "tag"
 		tarsFlag  = "tars"
 		tokenFlag = "token"
 	)
-	changelogCmd.PersistentFlags().StringVar(&changelogOpts.branch, "branch", git.Master, "The target release branch. Leave it default for non-patch releases.")
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.bucket, "bucket", "kubernetes-release", "Specify gs bucket to point to in generated notes")
+	changelogCmd.PersistentFlags().StringVar(&changelogOpts.tag, tagFlag, "", "The version tag of the release, for example v1.17.0-rc.1")
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.tars, tarsFlag, "", "Directory of tars to sha512 sum for display")
 	changelogCmd.PersistentFlags().StringVarP(&changelogOpts.token, tokenFlag, "t", "", "GitHub token for release notes retrieval")
 	changelogCmd.PersistentFlags().StringVarP(&changelogOpts.outputDir, "output", "o", os.TempDir(), "Output directory for the generated files")
 
-	if err := changelogCmd.MarkPersistentFlagRequired(tokenFlag); err != nil {
```

Comment:
the `tokenFlag` is not more required?

Reply:
No, we don't need the token if we do not generate notes, like when downloading it from GitHub. The release notes generator will complain as well if the token is not provided. :)

Reply:
thanks for the clarification